website/templatetags/permission_tags.py

```diff
@@ -2,22 +2,57 @@
 
 register = template.Library()
 
+
 def can_edit(user, obj):
     try:
         if user == obj.user or user.id == obj.uid:
-           return True
+            return True
         else:
-           return False
-    except:
+            return False
+    except BaseException:
         return False
+
+
 register.filter(can_edit)
 
+
 def is_moderator(user):
     try:
         if user.groups.count() > 0:
             return True
         else:
             return False
-    except:
+    except BaseException:
         return False
+
+
 register.filter(is_moderator)
+
+
+def comment_id(answer):
+    return "comment" + str(answer.id)
+
+
+register.filter(comment_id)
+
+
+def havenot_comments(answer):
+    comment = answer.answercomment_set.filter(is_active=True)
```

Comment:
Suggestion:
return answer.answercomment_set.filter(is_active=True).exists()

Also, function name can be better and consistent like underscores between the words.

Reply:
done